sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryTableRowIterator.java

```diff
@@ -36,6 +36,7 @@
 import com.google.api.services.bigquery.model.JobConfiguration;
```

Comment:
Fixed, diff becomes: (because of useLegacySql)
<pre>
< public class BigQueryTableRowIterator implements AutoCloseable {
---
> class BigQueryTableRowIterator implements AutoCloseable {
95,96d93
<   // Whether to use the BigQuery legacy SQL dialect..
<   private final boolean useLegacySql;
104c101
<       Bigquery client, boolean flattenResults, boolean useLegacySql) {
---
>       Bigquery client, boolean flattenResults) {
110d106
<     this.useLegacySql = useLegacySql;
119c115
<     return new BigQueryTableRowIterator(ref, null, ref.getProjectId(), client, true, true);
---
>     return new BigQueryTableRowIterator(ref, null, ref.getProjectId(), client, true);
124,126c120
<    * specified query in the specified project with useLegacySql set to True.
<    *
<    * @deprecated use {@link #fromQuery(String, String, Bigquery, Boolean, Boolean)}.
---
>    * specified query in the specified project.
128d121
<   @Deprecated
131,140d123
<     return fromQuery(query, projectId, client, flattenResults, null /* useLegacySql */);
<   }
< 
<   /**
<      * Constructs a {@code BigQueryTableRowIterator} that reads from the results of executing the
<      * specified query in the specified project.
<      */
<   public static BigQueryTableRowIterator fromQuery(
<       String query, String projectId, Bigquery client, @Nullable Boolean flattenResults,
<       @Nullable Boolean useLegacySql) {
145,146c128
<         MoreObjects.firstNonNull(flattenResults, Boolean.TRUE),
<         MoreObjects.firstNonNull(useLegacySql, Boolean.TRUE));
---
>         MoreObjects.firstNonNull(flattenResults, Boolean.TRUE));
166c148
<         // DataflowPipelineRunner.
---
>         // DataflowRunner.
261c243
<       return AvroUtils.formatTimestamp((String) v);
---
>       return BigQueryAvroUtils.formatTimestamp((String) v);
333c315
<         // To match that behavior, BigQueryTableRowiterator, and the DirectPipelineRunner,
---
>         // To match that behavior, BigQueryTableRowiterator, and the DirectRunner,
437,438d418
<     queryConfig.setUseLegacySql(useLegacySql);

</pre>